release.nix

```diff
@@ -16,25 +16,18 @@ rec {
     with import nixpkgs { inherit system; };
 
     python2Packages.buildPythonApplication rec {
-      name = "nixops-aws-${version}";
+      pname = "nixops-aws";
+      inherit version;
       namePrefix = "";
 
       src = ./.;
 
       prePatch = ''
-        for i in setup.py; do
-          substituteInPlace $i --subst-var-by version ${version}
-        done
+        substituteAllInPlace setup.py
```

Comment:
this lacks `--subst-var-by version ${version}` right ?

Reply:
`substituteAllInPlace` replaces every instance of `@foo@` with the value of `$foo`, if set.  Since `version` is set this does the same thing in less code.  If in doubt we can add a `cat setup.py` at the end of our patch phase to verify:

```
λ brh nixops-aws →  nix-build release.nix -A build.x86_64-linux
these derivations will be built:
  /nix/store/ajy200czzl0f8wbsqzb1j14fcl9dmlaq-nixops-aws-1.7pre0_abcdef.drv
building '/nix/store/ajy200czzl0f8wbsqzb1j14fcl9dmlaq-nixops-aws-1.7pre0_abcdef.drv'...
...
patching sources
from distutils.core import setup


setup(name='nixops-aws',
      version='1.7pre0_abcdef',
      description='NixOS cloud deployment tool, but for aws',
      url='https://github.com/NixOS/nixops-aws',
      # TODO: add author
      author='',
      author_email='',
      packages=['nixopsaws', 'nixopsaws.data', 'nixopsaws.resources', 'nixopsaws.backends'],
      entry_points={'nixops': ['aws = nixopsaws.plugin']},
      py_modules=['plugin']
)
configuring
no configure script, doing nothing
building
...
```

Reply:
great, I didn't know that. Thanks